src/components/YieldPools/ListItem.tsx

```diff
@@ -148,7 +154,9 @@ const ListItem = ({ farm, setSharedPoolAddress }: ListItemProps) => {
   const balance = useTokenBalance(pairAddressChecksum)
   const staked = useStakedBalance(farm.fairLaunchAddress, farm.pid)
   const rewardUSD = useFarmRewardsUSD(farmRewards)
-  const { mixpanelHandler } = useMixpanel()
+
+  const isNotDeposited = !balance.value.gt(0)
```

Comment:
classic farm dont have deposit step, it's only stake. So I suggest that this variable should be `canStake` or isAbleToStake`